google/cloud/pubsub/internal/default_pull_lease_manager.cc

```diff
@@ -98,8 +99,8 @@ future<Status> DefaultPullLeaseManager::ExtendLease(
       [stub = std::move(stub), deadline = now + extension, clock = clock_,
        impl = impl_](auto cq, auto context, auto options, auto const& request) {
         if (deadline < clock->Now()) {
-          return make_ready_future(
-              Status(StatusCode::kDeadlineExceeded, "lease already expired"));
+          return make_ready_future(internal::FailedPreconditionError(
```

Comment:
I think `internal::DeadlineExceededError`

Reply:
Agh, ok I mixed up some of the find/replace

Reply:
Done